deps.sh

```diff
@@ -199,6 +199,7 @@ check_debian_pkgs () {
     pkgconf            # build system
     zstd               # build system
     gzip               # build system
+    libbz2-dev         # blocksort
```

Comment:
Why did you need to include this dependency? Firedancer should compile fine without it